uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/crm/app.form.absenceperiod.js

```diff
@@ -44,6 +44,41 @@ define([
       this._validateOrder();
     };
 
+    /**
+     * Prompts the user via a modal to calculate entitlement for
+     * a new absence period
+     *
+     * @private
+     */
+    AbsencePeriod.prototype._showEntitlementCalculationPrompt = function () {
+      var confirmationMessage = 'The system will now update the staff members ' +
```

Comment:
No need to split, just put it on the next line:

```js
var confirmationMessage =
  'The system will now update the staff members leave entitlement';
```
or even as an exception:

```js
var confirmationMessage = 'The system will now update the staff members leave entitlement';
```

Please note, we are not **strictly** following the symbols amount rule, this is mostly for us to keep the code readable. This is just a long string, you will not gain any more info by reading it whole or just a half. So it's fine to keep it one-liner.